docker-compose/nginx/viewer/app-config.js

```diff
@@ -1,7 +1,32 @@
 window.config = {
-  // default: '/'
   routerBasename: '/',
-  extensions: ['cornerstone', 'dicom-p10-downloader'],
+  extensions: [
+    {
+      id: 'cornerstone',
+      getViewportModule: function() {
+        return [
+          {
+            name: 'cornerstone',
+            component: 'cornerstoneViewportComponent',
+          },
+        ];
+      },
+      getSopClassHandlerModule: function() {
+        return [
+          {
+            name: 'cornerstone',
+            sopClassUIDs: [
+              '1.2.840.10008.5.1.4.1.1.2', // CT Image Storage
```

Comment:
Hi @pierrehenri-dauvergne, excellent, this PR looks very promising :-)! Please add more SOP Class UIDs here for other modalities supported by Shanoir: PT, NM.
'1.2.840.10008.5.1.4.1.1.2.1' , // Enhanced CT Image Storage
'1.2.840.10008.5.1.4.1.1.4.1', // Enhanced MR Image Storage
'1.2.840.10008.5.1.4.1.1.4.3', // Enhanced MR Color Image Storage
'1.2.840.10008.5.1.4.1.1.130', // Enhanced PET Image Storage
'1.2.840.10008.5.1.4.1.1.20', // Nuclear Medicine Image Storage